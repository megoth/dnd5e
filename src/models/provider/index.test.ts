import { getProviders } from "./index";

describe("getProviders", () => {
  it("returns a list with info about IdPs", () => {
    expect(getProviders()).toMatchSnapshot();
  });
});
