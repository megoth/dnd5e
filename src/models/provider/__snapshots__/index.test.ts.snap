// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getProviders returns a list with info about IdPs 1`] = `
Array [
  Object {
    "iri": "https://broker.pod.inrupt.com/",
    "label": "Inrupt Pod Spaces",
    "signupIri": "https://signup.pod.inrupt.com/",
  },
  Object {
    "iri": "https://solidcommunity.net/",
    "label": "solidcommunity.net",
    "signupIri": "https://solidcommunity.net/register",
  },
  Object {
    "iri": "https://inrupt.net/",
    "label": "inrupt.net",
    "signupIri": "https://inrupt.net/register",
  },
]
`;
